File: frontend/EduConnect/src/app/models/course/course-tutor-controller/get-course-teaching-resource-response.ts
export interface GetCourseTeachingResourceResponse {
  courseTeachingResourceId: string;
  title: string;
  description: string;
  resourceUrl: string | null;
  fileName: string | null;
  contentType: string | null;
  fileSize: number | null;
  createdAt: string;
}

File: backend/DTOs/GetCourseTeachingResourceResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace EduConnect.DTOs
{
    public class GetCourseTeachingResourceResponse
    {
        public Guid CourseTeachingResourceId { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public string? ResourceUrl { get; set; }

        public string? FileName { get; set; }
        public string? ContentType { get; set; }
        public long? FileSize { get; set; }

        public DateTime CreatedAt { get; set; }
    }
}